lib/core/text/abstract_text.nit

```diff
@@ -617,6 +620,17 @@ abstract class Text
 				b.append("\\\'")
 			else if c == '\\' then
 				b.append("\\\\")
+			else if c == '?' then
+				# Escape if it is the last question mark of a ANSI C trigraph.
+				var j = i + 1
+				if j < length then
+					var next = chars[j]
+					# We ignore `??'` because it will be escaped as `??\'`.
+					if ['=', '/', '(', ')', '!', '<', '>', '-'].has(next) then
```

Comment:
Sorry. I was (stupidly) assuming an optimization here. I will replace it by a bunch of `or`s.